docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/ecommerce-tracking/index.md

```diff
@@ -0,0 +1,702 @@
+| id          | Y        | string          | The unique promotion identifier.                   |
+| name        | N        | string          | Promotion name.                                    |
+| productIds  | N        | list of strings | IDs of products in the promotion.                  |
+| position    | N        | integer         | Index of the promotion in a list such as a banner. |
+| creativeId  | N        | string          | Identifier for the promotion creative.             |
+| type        | N        | string          | Type of promotion.                                 |
+| slot        | N        | string          | Where the promotion was displayed.                 |
+
+</details>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.ecommerce/promotion/jsonschema/1-0-0`.
+
+## Global ecommerce entities Screen/Page and User
+
+Use these APIs to add ecommerce context information to every subsequent event tracked.
+
+:::note
+These entities will be added to **all** events, not just ecommerce ones.
+:::
+
+### EcommerceUser entity
+
+To set an Ecommerce User entity you can use the `setEcommerceUser` method with the following attributes:
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+Coming soon!
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+Snowplow.defaultTracker?.ecommerce.setEcommerceUser("userId12345")
+
+// setting EcommerceUser again will replace the original entity
+Snowplow.defaultTracker?.ecommerce.setEcommerceUser("userId67890")
+
+// remove the saved properties and stop the User entity being added
+Snowplow.defaultTracker?.ecommerce.removeEcommerceUser()
+```
+
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+Snowplow.getDefaultTracker().getEcommerce().setEcommerceUser("userId12345");
+
+// setting ScreenType again will replace the original entity
+Snowplow.getDefaultTracker().getEcommerce().setEcommerceUser("userId67890");
+
+// remove the saved properties and stop the Page entity being added
+Snowplow.getDefaultTracker().getEcommerce().removeEcommerceUser();
+```
+
+  </TabItem>
+</Tabs>
+
+<details>
+    <summary>User entity properties</summary>
+
+| Request Key | Required | Type/Format | Description                  |
+|-------------|----------|-------------|------------------------------|
+| id          | Y        | string      | The unique user identifier.  |
+| isGuest     | N        | boolean     | Whether the user is a guest. |
+| email       | N        | string      | User email address.          |
+
+</details>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.ecommerce/user/jsonschema/1-0-0`.
+
+### EcommerceScreen (Page) entity
+
+:::note
+The `setEcommerceScreen` method adds a `Page` (rather than `Screen`) entity to all events, for consistency with web tracking.
+:::
+
+To set a Page entity you can use the `setScreenType` method:
```

Comment:
Nit: It's not really obvious to me what is the entity for. Can we just say with one sentence how would one use the entity? For example to describe the current screen of the eshop where the ecommerce actions happen or something like that.